### PR TITLE
Adding schema compare UX support for external streams/streaming jobs

### LINKED ISSUE #12816: External Streams and External Streaming Jobs missing from include/exclude object types in schema compare

### PR DESCRIPTION
**Pull request by Benjin**
Fixes #12816

### CODE REVIEW

**Review by kisantia — APPROVED** [review]

**Comment by Benjin** [comment]
I didn't see anything in extension tests or SqlToolsService that needs updating as a result of this (and it's working in manual testing).  @kisantia anything I missed?

**Comment by kisantia** [comment]
should be fine as long as existing tests are passing and manual testing verified this works now

File: extensions/mssql/src/mssql.d.ts [hunk 1]
```diff
@@ -288,7 +291,9 @@ export const enum SchemaObjectType {
 	ServerAuditSpecifications = 63,
 	ServerRoleMembership = 64,
 	ServerRoles = 65,
-	ServerTriggers = 66
+	ServerTriggers = 66,
```
Comment (udeeshagautam) [inline extensions/mssql/src/mssql.d.ts:294]: the number match dacfx right :)

**Review by udeeshagautam — APPROVED** [review]

File: extensions/mssql/src/mssql.d.ts (hunk 1 above)
Comment (Benjin) [reply extensions/mssql/src/mssql.d.ts:294]: Yup.  Or at least the five or six ones I randomly spot-checked do! :joy:

**Pull request merged**